packages/sdk/src/ApiLimitsHandler.ts

```diff
@@ -0,0 +1,35 @@
+import { RequestController } from './RequestController'
+
+export class ApiLimitsHandler {
+	constructor(private readonly requestController: RequestController) {}
+
+	private getLastResponseHeadersFromController(): Record<string, string> {
+		return this.requestController.getLastResponseHeaders()
+	}
+
+	private isLastResponseHeadersSet(): boolean {
```

Comment:
Please call this `areLimitsAvailable` and make it public.
Add info in docs about this method and that it should be always called before calling `getRateLimit` or `getRateLimitRemaining`